test/unit/services/storage/models.spec.ts

```diff
@@ -343,7 +343,7 @@ describe('Models', () => {
         { provider: 'provider2', totalCapacity: 100, peerId: '2' }
       ])
       await Agreement.bulkCreate([
-        { agreementReference: '123', size: 10.0000000000000001, offerId: 'provider1' },
```

Comment:
I removed the decimals as we are not storing decimals in the DB so was not a real test